css-layout-classes/styles.css

```diff
@@ -19,6 +19,6 @@
   flex-basis: 66%;
 }
 
-img {
+.column-one-fourth {
   max-width: 100%;
 }
```

Comment:
If it's a `column-one-fourth`, it shouldn't take up 100% width right? It should only take up 1/4th or `25%`